### PR TITLE
feat: working video version picker

### LINKED ISSUE #1931: Version Selector is not working
### Description of the bug

Version selector for multiple versions available movies are not working. Right now available versions are showing as a single line.

### Steps to reproduce

1. Make sure you have a movie/episode with multiple versions
2. Try to select second version

### Expected behavior

Version selector should work and media should be able to switch

### Logs

_No response_

### Screenshots

Screenshot from 2023-04-07 11-50-36

### Platform

Linux

### Browser

Firefox

### Jellyfin server version

10.8.9

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ThibaultNocchi**
fix #1931 

Thanks to @HazilMohamed for catching the issue and implementing the v-select fix :)

### CODE REVIEW

File: frontend/src/pages/item/_itemId/index.vue [hunk 1]
```diff
@@ -288,6 +294,8 @@ onMounted(async () => {
     })
   ).data;
 
+  selectSources.value = getItemizedSelect(item.value.MediaSources ?? []);
```
Comment (ferferga) [inline frontend/src/pages/item/_itemId/index.vue:297]: The computed should live below all the declarations so we avoid reference errors.

**Review by ferferga — CHANGES_REQUESTED** [review]

File: frontend/src/pages/item/_itemId/index.vue (hunk 1 above)
Comment (ThibaultNocchi) [reply frontend/src/pages/item/_itemId/index.vue:297]: I think I'll just re-make the onMounted part to avoid variables initialisation like this

Comment (ThibaultNocchi) [reply frontend/src/pages/item/_itemId/index.vue]: @ferferga fixed the order by removing the onMounted logic

File: frontend/src/pages/item/_itemId/index.vue [hunk 2]
```diff
@@ -246,13 +247,24 @@ import {
...
+const { itemId } = route.params as { itemId: string };
+
+const item = ref(
+  (
+    await remote.sdk.newUserApi(getUserLibraryApi).getItem({
+      userId: remote.auth.currentUserId ?? '',
+      itemId
+    })
+  ).data
+);
```
Comment (ferferga) [inline frontend/src/pages/item/_itemId/index.vue:265]: Do we need this to be a ref?

File: frontend/src/pages/item/_itemId/index.vue [hunk 3]
```diff
@@ -246,13 +247,24 @@ import {
 import { getUserLibraryApi } from '@jellyfin/sdk/lib/utils/api/user-library-api';
 import { getBlurhash } from '@/utils/images';
 import { getItemDetailsLink, getMediaStreams } from '@/utils/items';
-import { getItemizedSelect } from '@/utils/forms';
+import { getItemizedSelect, VSelectItem } from '@/utils/forms';
```
Comment (ferferga) [inline frontend/src/pages/item/_itemId/index.vue:250]: Fix the lint issue from here, please

**Review by ferferga — CHANGES_REQUESTED** [review]

File: frontend/src/pages/item/_itemId/index.vue (hunk 2 above)
Comment (ThibaultNocchi) [reply frontend/src/pages/item/_itemId/index.vue]: Indeed, doesn't seem so. I removed the ref.

File: frontend/src/pages/item/_itemId/index.vue (hunk 3 above)
Comment (ThibaultNocchi) [reply frontend/src/pages/item/_itemId/index.vue]: Good catch, I must have been tired I forgot many of them ^^'

**Comment by ThibaultNocchi** [comment]
@ferferga Addressed your comments

**Review by ferferga — APPROVED** [review]

**Pull request merged**